readme.md

```diff
@@ -38,7 +41,7 @@ Describe the differences between a SQL and NoSQL DB, and when you might use each
 
 Your Answer:
 ```text
-
+SQL databases store data in tables and each row of the table is for one data point. NoSQL stores data in objects using key value pairs. The SQL databases are less flexible than NoSQL databases but better at mapping complex relationships such as many-to-many relationships. Any changes to a SQL database would need to start with the schema and adding/editin/removing columns from the tables. A feature of NoSQL databases is that they are adaptable to changes anytime and anywhere.
```

Comment:
:confetti_ball: Excellent! I've been saying that I wouldn't say better at mapping complex relationships as you can do some really complex stuff with NoSQL but that relational database make kinds of relationships easier to model and fast. We said in class though that SQL database are better at mapping complex relationships. Better just seems like a really broad word to me.